sklearn/metrics/_pairwise_distances_reduction/_base.pxd.tp

```diff
@@ -1,6 +1,6 @@
 from cython cimport final
 
-from ...utils._typedefs cimport intp_t, float64_t
+from sklearn.utils._typedefs cimport intp_t, float64_t
```

Comment:
This change is unrelated and already implemented in your previous PR. Please revert it.